src/components/TaskCards.tsx

```diff
@@ -29,6 +43,13 @@ export default function TaskCards() {
             primary={task}
             secondary={`Expires at: ${expiresAt}`}
           />
+          <IconButton
+            aria-label="delete"
+            size="small"
+            onClick={() => deleteOnClick(id)}
```

Comment:
the same, separate func

Reply:
I don't really understand how to make a new func in this case (with passing id parameter)

Reply:
done